projects/tuist/fixtures/app_with_plugins/Tuist/Config.swift

```diff
@@ -4,6 +4,6 @@ let config = Config(
     plugins: [
         .local(path: .relativeToManifest("../../LocalPlugin")),
         // TODO: Change back to tuist organization
-        .git(url: "https://github.com/fortmarek/ExampleTuistPlugin", tag: "0.4.0"),
+        .git(url: "https://github.com/woohyunjin06/ExampleTuistPlugin", tag: "3.1.0"),
```

Comment:
Can't we just do the change in `https://github.com/tuist/ExampleTuistPlugin`?

Also, why did you need to change the executable below?

Reply:
I've confirmed that this url does not affect the new test case, and I have updated the repository url.

The reason for change the executable is that test case I created in this PR use the updated `create-file` command to check whether environment variable is defined or not.